flutter/macos/sentry_flutter.podspec

```diff
@@ -0,0 +1,22 @@
+#
+# To learn more about a Podspec see http://guides.cocoapods.org/syntax/podspec.html.
+# Run `pod lib lint sentry_flutter.podspec` to validate before publishing.
+#
+Pod::Spec.new do |s|
+  s.name             = 'sentry_flutter'
+  s.version          = '0.0.1'
+  s.summary          = 'A new flutter plugin project.'
```

Comment:
lets copy summary, authors etc from the podpsec from the iOS plugin

Reply:
Yep, sure